experimental/dds/tree2/src/test/domains/json/twitter.ts

```diff
@@ -22,30 +22,73 @@ import {
  * json benchmarks twitter.json - https://raw.githubusercontent.com/serde-rs/json-benchmark/master/data/twitter.json
  */
 // Shared tree keys that map to the type used by the Twitter type/dataset
-export const Twitter = {
-	SharedTreeFieldKey: {
-		statuses: brand<FieldKey>("statuses"),
-		retweetCount: brand<FieldKey>("retweet_count"),
-		favoriteCount: brand<FieldKey>("favorite_count"),
-	},
+export const TwitterKey = {
+	statuses: brand<FieldKey>("statuses"),
+	retweetCount: brand<FieldKey>("retweet_count"),
+	favoriteCount: brand<FieldKey>("favorite_count"),
 };
 
-export interface Twitter {
-	statuses: TwitterStatus[];
-	search_metadata: {
-		completed_in: number;
-		max_id: number;
-		max_id_str: string;
-		next_results: string;
-		query: string;
-		refresh_url: string;
-		count: number;
-		since_id: number;
-		since_id_str: string;
+/* eslint-disable @rushstack/no-new-null */
+export interface TwitterUser {
+	id: number;
+	id_str: string;
+	name: string;
+	screen_name: string;
+	location: string;
+	description: string;
+	url: string | null;
+	entities: {
```

Comment:
The next set of changes is just reorder the interfaces so types are declared before they are referenced.

This is required when converting to schema, since these will become variables.